.github/workflows/model-run.yml

```
@@ -0,0 +1,57 @@
+name : Sipnet Model Run
+on :
+  pull_request:
+
+  # allow manual triggering
+  workflow_dispatch:
+
+  schedule:
+    # run Thursday 4:30 AM UTC
+  - cron: '30 4 * * 4'    
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+
+    env:
+      GITHUB_PAT: ${{ secrets.GITHUB_TOKEN }}
+
+    steps:
+    - name: Checkout source code
+      uses: actions/checkout@v4
+
+    - name: Set up Docker Compose
+      run: |
+          sudo apt-get update
+          sudo apt-get -y install docker-compose
+
+    - name: Build Containers
+      run: |
+        cp docker/env.example .env
+        echo "COMPOSE_PROJECT_NAME=pecan" >> .env
+        echo "PECAN_VERSION=develop" >> .env
+        echo "UID=$(id -u)" >> .env
+        echo "GID=$(id -g)" >> .env
+        
+        docker-compose up -d postgres
+        docker run --rm --network pecan_pecan pecan/db
+        
+        docker-compose run bety user guestuser guestuser "Guest User" guestuser@example.com 4 4
+        docker-compose run bety user carya illinois "Carya Demo User" carya@example.com 1 1
```

We can skip creating a guest user for headless runs
```suggestion
        docker compose run --rm bety user carya illinois "Carya Demo User" carya@example.com 1 1
```